contrib/onscripter-en/patches/smpeg_register.patch

```diff
@@ -0,0 +1,728 @@
+iso c++17 does not allow register storage class specifier, so yank it indiscriminately
```

Comment:
just link the system one unless this is patched

otherwise -std=c++11 the subproject in buildsystem and you can drop the entire patch; clang just defaults to 17 when unspecified, this is still valid in 11